tests/SpecTests/ClientSideEncryptionSpecTest.php

```diff
@@ -168,9 +168,12 @@ public function provideTests()
      */
     public function testDataKeyAndDoubleEncryption(Closure $test)
     {
-        $client = new Client(static::getUri());
+        $this->setContext(Context::fromClientSideEncryption((object) [], 'db', 'coll'));
+        $client = $this->getContext()->getClient();
 
-        $client->selectCollection('keyvault', 'datakeys')->drop();
+        // This empty call ensures that the key vault is dropped with a majority
+        // write concern
+        $this->insertKeyVaultData([]);
```

Comment:
Looking at `insertKeyVaultData()`, it appears to no-op if its argument is empty.